backend/de.metas.business/src/main/java/de/metas/costing/methods/AveragePOCostingMethodHandler.java

```diff
@@ -165,44 +170,51 @@ private CostDetailCreateResult createCostDetailAndAdjustCurrentCosts(final CostD
 		if (isInboundTrx || request.isReversal())
 		{
 			// Seed/initial costs import
-			if (request.getDocumentRef().isInventoryLine() && request.getQty().signum() == 0)
+			final CostAmount requestAmt = request.getAmt();
+
+			if (request.getDocumentRef().isInventoryLine())
```

Comment:
I would keep this condition because I don't know the implications in other kinds of documents ( see line 197 and below)